app.json

```
@@ -9,8 +9,11 @@
     "userInterfaceStyle": "light",
     "splash": {
       "image": "./assets/splash.png",
-      "resizeMode": "cover",
-      "backgroundColor": "#050F77"
+      "backgroundColor": "#050F77",
+      /*resizeMode - native work only for android
+      https://docs.expo.dev/develop/user-interface/splash-screen/#custom-configuration-for-android-and-ios
+      **/
```

guess app.json is secretly a jsonc/json5 file? 😄 

```suggestion
      // resizeMode - native work only for android: https://docs.expo.dev/develop/user-interface/splash-screen/#custom-configuration-for-android-and-ios
```

---

@achou11 XD what do you mean :D ?

---

mostly just joking how the file extension doesn't match how it's treated - comments are not allowed in json files but if expo doesn't care, then it's treating this file as jsonc/json5, which supports comments but technically should be using a different file extension name 😄 

suggestion here is to clean up the comment formatting (basically make it a single line comment)

---

oh :D it's work, that's why I put them :)

---

Eventually this will be moved to app.config.js file anyway 😁 I'll open PR for it tomorrow.